### PR TITLE
Add an option for formatting the results

### PR DESCRIPTION
**Pull request by Ardasak**
This PR offers formatting the results by removing dots and commas (optional).

### CODE REVIEW

**Comment by Ardasak** [comment]
@egeakman we could delete the last commit.

File: src/functions.py [hunk 1]
```diff
@@ -15,8 +15,10 @@
     "doz4turkiyeortalamasi",
     "asidozuguncellemesaati",
 }
-def remove_dots_and_commas(string):
-    return string.replace(".", "").replace(",", "")
+def remove_dots_and_commas(string, formatted = False):
```
[the remark this answers is not in the document]
Comment (egeakman) [reply src/functions.py]: And also, we should check if the type of ``string`` is ``str``.

**Review by egeakman — CHANGES_REQUESTED** [review]

**Comment by egeakman** [comment]
I will add tests for arg: ``formatted``

**Review by egeakman — APPROVED** [review]
Looks good now.

**Pull request merged**